RocketApp/Features/Tests/RocketListTests/RocketListTests.swift

```diff
@@ -1,8 +1,63 @@
 @testable import RocketList
+import Combine
 import XCTest
+import RocketsClient
+import ComposableArchitecture
+import NetworkClientExtensions
 
-final class FeaturesTests: XCTestCase {
-  func testExample() throws {
-    XCTAssertTrue(true)
+final class RocketListTests: XCTestCase {
+  var store = TestStore(initialState: RocketListCore.State(), reducer: RocketListCore())
```

Comment:
Proc je to `var`?

Reply:
Jinak ja sem spis priznivce toho, aby si kazdej test inicializoval ten `TestStore` sam.